### PR TITLE
[IMP] theme_graphene: revamp the theme

### PR DESCRIPTION
**Pull request by mano-odoo**
task-4178075
Part of task-4177975

### CODE REVIEW

**Review by robinlej — CHANGES_REQUESTED** [review]

File: theme_graphene/views/customizations.xml [hunk 1]
```diff
@@ -226,38 +254,74 @@
...
+        <attribute name="class" add="o_cc o_cc2 pt64 pb64" remove="pt48 pb48" separator=" "/>
     </xpath>
 </template>
 
-<template id="configurator_s_comparisons" inherit_id="website.configurator_s_comparisons" name="Graphene s_comparisons">
+<!-- ======== NUMBERS ======== -->
+<template id="s_numbers" inherit_id="website.s_numbers" name="Graphene s_numbers">
     <xpath expr="//section" position="attributes">
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/06_001","flip":["x"]}</attribute>
+        <attribute name="class" add="oe_img_bg o_bg_img_center pb56 pt56 s_parallax_no_overflow_hidden parallax s_parallax_is_fixed o_cc3" remove="pb80 pt80 o_cc1" separator=" "/>
```
[the remark this answers is not in the document]
Comment (mano-odoo) [reply theme_graphene/views/customizations.xml:264]: I think the diff is doing something strange. I am not touching these lines, it seems to weirdly conflict with the new `s_numbers_grid` template I added under 😕 I will still seize the opportunity to add the data-scroll-background-ratio 😄 
image

File: theme_graphene/views/customizations.xml [hunk 2]
```diff
@@ -4,21 +4,24 @@
 <!-- ======== COVER ======== -->
 <template id="s_cover" inherit_id="website.s_cover" name="Graphene s_cover">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt160 pb240 s_parallax_no_overflow_hidden o_cc3" remove="pt232 pb232 s_parallax_is_fixed s_parallax o_cc4" separator=" "/>
+        <attribute name="class" add="pt160 pb192 s_parallax_no_overflow_hidden o_cc5" remove="pt232 pb232 s_parallax_is_fixed s_parallax o_cc4" separator=" "/>
```
[the remark this answers is not in the document]
Comment (mano-odoo) [reply theme_graphene/views/customizations.xml:7]: I am just replacing the padding and the color with `o_cc5` on this line so it was already there, but indeed it highlighted an issue. I don't think we want a parallax at all here, I will replace as suggested 😄

File: theme_graphene/views/customizations.xml [hunk 3]
```diff
@@ -45,29 +51,27 @@
...
-        <h3>Unique experiences to drive engagement</h3>
+        <h2>Unique experiences to drive engagement</h2>
         <div class="s_hr pt16 pb0" data-snippet="s_hr" data-name="Separator">
             <hr class="w-100 mx-auto" style="border-top-color: rgba(0, 0, 0, 0) !important;"/>
         </div>
     </xpath>
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_image_text_device_perspective/web_editor/devices/iphone_3d_portrait_02.svg</attribute>
+        <attribute name="data-shape">web_editor/devices/iphone_3d_portrait_02</attribute>
+    </xpath>
```
Comment (robinlej) [inline theme_graphene/views/customizations.xml:63]: The editor doesn't work properly. You can't modify the shape and a few other options. Maybe you just need to add a few more attributes for it to work :thinking: (cf. something similar is done in `theme_odoo_experts/views/snippets/s_image_text.xml`)

| wrong | right |
| --- | --- |
| image | image |

File: theme_graphene/views/customizations.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (robinlej) [reply theme_graphene/views/customizations.xml:271]: Oh yeah, weird. The diff is ok if I check it in my terminal, so it's a github issue. :shrug:

File: theme_graphene/views/customizations.xml (hunk 3 above)
Comment (mano-odoo) [reply theme_graphene/views/customizations.xml]: Good catch! 

It took me some time to figure out, we need a `s_image_text_device` record for it to work. I guess there is some black magic somewhere in the code which shows the properties of this record, while visually showing the `_perspective` image version of it in the device shape for the first load. 🤯 

It should be fine now 😄

Comment (robinlej) [reply theme_graphene/views/customizations.xml]: Oh well done, I had no idea :exploding_head: It's good to know. I found the JS responsible for it in web_editor:

```js
let match = img.src.match(/\/web_editor\/image_shape\/(\w+\.\w+)/);
if (img.dataset.shape && match) {
    match = match[1];
    if (match.endsWith("_perspective")) {
        // As an image might already have been modified with a
        // perspective for some customized snippets in themes. We need
        // to find the original image to set the 'data-original-src'
        // attribute.
        match = match.slice(0, -12);
    }
    return this._loadImageInfo(`/web/image/${encodeURIComponent(match)}`);
}
```

**Comment by sobo-odoo** [comment]
@robodoo delegate=robinlej

**Comment by robinlej** [comment]
@robodoo r+

**Pull request closed without merging**